config/initializers/delayed_job.rb

```diff
@@ -1,5 +1,6 @@
 require 'delayed_job'
 Delayed::Worker.max_attempts = 3
 Delayed::Worker.max_run_time = 5.minutes
+Delayed::Worker.logger = Logger.new(File.join(Rails.root, 'log', 'delayed_job.log'))
```

Comment:
Could do `Rails.root.join("log/delayed_job.log")` ...if `Logger.new` can take a `Pathname`. If not then just `to_s` it.